truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/impl/Accessor.java

```diff
@@ -277,6 +278,7 @@ protected Debugger createDebugger(Object vm, Instrumenter instrumenter) {
     @SuppressWarnings("unused")
     protected Closeable executionStart(Object vm, int currentDepth, Debugger debugger, Source s) {
         vm.getClass();
```

Comment:
That line isn't subject of this pull request. Feel free to start your own if you care that much.